docs/manual-EN/import_data/nebula-importer-updated.md

```diff
@@ -243,7 +244,7 @@ _src,_dst,_ranking,prop1,prop2
 
 `_src` and `_dst` represent edge source and destination vertex id. `_ranking` column is value of edge ranking.
```

Comment:
`_src` and `_dst` represent the edge's source vertex ID and dest vertex ID.  `_ranking` is the edge's weight.